spec/unit/module_repository/forge_spec.rb

```diff
@@ -19,15 +34,33 @@
     expect(forge.forge).to eq 'forge.example.local'
   end
 
-  describe "and the expected version is :latest", :vcr => true do
+  describe "and the expected version is :latest" do
     subject(:forge) { described_class.new }
 
     before do
-      forge.conn.builder.insert_before(Faraday::Adapter::NetHttp, VCR::Middleware::Faraday)
+      boolean_body = <<-JSON
+      {"releases": [
+          {"version": "1.0.1"},
+          {"version": "1.0.0"},
+          {"version": "0.9.0"}
+        ]
+      }
+      JSON
+
+      spotty_body = <<-JSON
+      {"releases": [
+          {"version": "0.4.0", "deleted_at": "some point"},
+          {"version": "0.3.0"}
+        ]
+      }
+      JSON
```

Comment:
Might be cleaner to define these as hashes and just call to_json on them? Heredoc is going to be full of leading whitespace.